website/pages/community/index.jsx

```diff
@@ -18,12 +18,12 @@ export default function CommunityPage() {
           {
             header: 'Community Forum',
             body:
-              '[Boundary Community Forum](https://discuss.hashicorp.com/c/boundary)',
+              '<a href="https://discuss.hashicorp.com/c/boundary">Boundary Community Forum</a>',
```

Comment:
`vertical-text-block-list` doesn't contain markdown processing anymore, so we need to pass it HTML. This could be done automatically in `getStaticProps` if we feel it'd be worth it for authoring convenience.